rmf_fleet_adapter/src/full_control/main.cpp

```diff
@@ -824,6 +824,9 @@ std::shared_ptr<Connections> make_fleet(
       if (!connections)
         return;
 
+      if (request_msg->fleet_name != fleet_name && !request_msg->fleet_name.empty())
```

Comment:
Sorry again for how nitpicky the style checker is, but this line seems to violate the 80-character limit. The style checker requires this to be changed to

```
      if (request_msg->fleet_name != fleet_name &&
      !request_msg->fleet_name.empty())
```

I think once that's fixed we'll be able to merge this. Thanks for bearing with our CI process :bow: